core/src/storage/impls/multi_version_merkle_patricia_trie/merkle_patricia_trie/cow_node_ref.rs

```diff
@@ -2,6 +2,20 @@
 // Conflux is free software and distributed under GNU General Public License.
 // See http://www.gnu.org/licenses/
 
+/// Load children merkles only when the number of uncached children nodes is
+/// above this threshold. Note that a small value will result in worse
+/// performance.
+const CHILDREN_MERKLE_UNCACHED_THRESHOLD: u32 = 8;
```

Comment:
Let's discuss more about why it's 8, rather than 2.

Reply:
Initial benchmark results show that a small value like 4 can cause more DB loads. I have no idea how it interacts with `CHILDREN_MERKLE_DEPTH_THRESHOLD` yet.